File: server/spring-api/src/main/java/no/uit/springapi/model/Person.java
package no.uit.springapi.model;

import lombok.*;

import javax.persistence.Column;
import javax.persistence.EnumType;
import javax.persistence.Enumerated;
import javax.persistence.MappedSuperclass;
import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@MappedSuperclass
public class Person extends BaseEntity {
    @Column(name="first_name")
    private String firstName;

    @Column(name="last_name")
    private String lastName;

    @Column(name="date_of_birth")
    private Date dateOfBirth;

    @Column(name="address")
    private String address;

    @Column(name="phone_number")
    private String phoneNumber;

    @Column(name="gender")
    @Enumerated(EnumType.STRING)
    private Gender gender;
}

File: server/api/API/Entities/Person.cs
using System;

namespace API.Entities
{
    public class Person
    {
        public int Id { get; set; }
        public DateTime DateOfBirth { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;
        public string Gender { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
    }
}